django/librarian/utils/process_engine.py

```diff
@@ -126,6 +126,7 @@ def guess_content_type(
         "application/csv",
         "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
         "officedocument.spreadsheetml.sheet",
+        "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
         "officedocument.wordprocessingml.document",
         "officedocument.presentationml.presentation",
```

Comment:
I'd say there's a decent chance this same issue will emerge with PPTX - could we add the similar variant for presentation here too?